### ISSUE TITLE
Configuration change resets bottom bar position

### ISSUE THREAD

**Issue by russellbanks** [Droppers/AnimatedBottomBar#35]
Selecting a different tab from the default and then changing the theme will reset the current tab selection. This means that it will be out of sync with the ViewPager. The tab selection should be maintained even after configuration changes - I have attached a video to show this.

https://user-images.githubusercontent.com/74878137/129940838-67af8c70-2974-434b-8976-5e6c42888fa7.mp4

**Comment by Droppers** [comment]
Yes makes sense, since onSaveInstanceState has not been implemented yet.

I will implement this as soon as I have time for it.
